packages/opentelemetry-exporter-console/src/ConsoleExporter.ts

```diff
@@ -0,0 +1,81 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { hrTimeToMicroseconds } from '@opentelemetry/core';
+import { ReadableSpan, SpanExporter } from '@opentelemetry/tracing';
+import { ExportResult } from '@opentelemetry/base';
+
+/**
+ * Simple console exporter for spans
+ */
+export class ConsoleExporter implements SpanExporter {
```

Comment:
We already have implemented `ConsoleSpanExporter`: https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-tracing/src/export/ConsoleSpanExporter.ts#L26

I'd like to use `_exportInfo` function there to log `span` data on the `console`. WDYT?

Reply:
@mayurkale22 I will refactor then the existing one and will move there the function and unit test